ext/random/random.stub.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/** @generate-class-entries */
+/** @generate-function-entries */
+
+namespace Random\NumberGenerator
+{
+    interface RandomNumberGenerator
+    {
+        /** 
+         * @tentative-return-type
```

Comment:
These tentative-return-type annotations should not be present in new code.

Reply:
Remove it.